### PR TITLE
Make sure that classpath.index file doesn't show up in build/classes/

### PR DESCRIPTION
**Pull request by PawelLipski**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -28,12 +28,12 @@ jobs:
           keys:
             # First, try finding a cache entry with the same set of libraries
             # that also comes from the same branch (so as to make the best use of Gradle compilation cache).
-            - gradle-deps-v5-{{ checksum "gradle/libs.versions.toml" }}-{{ .Branch }}
+            - gradle-deps-v27-{{ checksum "gradle/libs.versions.toml" }}-{{ .Branch }}
```
Comment (mkondratek) [inline .circleci/config.yml:31]: y 27 not 6?

File: build.gradle [hunk 2]
```diff
@@ -341,6 +341,11 @@ allprojects {
       // For the frontend subprojects we only use gradle-intellij-plugin to provide dependencies,
       // but don't want the associated tasks to be available; they should only be available in the root project.
       tasksAfter.each { it.enabled = false }
+      // The only task (as of gradle-intellij-plugin v1.7.0, at least) that needs to be enabled in all IntelliJ-aware modules
+      // is `classpathIndexCleanup`, to avoid caching issues caused by `classpath.index` file
+      // showing up in build/classes/ and build/resources/ directories.
+      // See https://github.com/JetBrains/gradle-intellij-plugin/issues/1039 for details.
```
Comment (mkondratek) [inline build.gradle:347]: 👍

Comment (mkondratek) [reply build.gradle:347]: isn't that ready (released)?

File: .circleci/config.yml (hunk 1 above)
Comment (PawelLipski) [reply .circleci/config.yml:31]: Doesn't harm tho

**Review by mkondratek — APPROVED** [review]

**Pull request merged**